Don't add weird non-MacPorts paths for freerdp.

Don't claim freerdp support is broken and disabled by default on macOS.
I don't know whether it's broken, but it does build by default and has
to be disabled explicitly. It doesn't build with MacPorts freerdp 2.x.
--- configure.ac.orig	2015-11-24 13:16:00.000000000 -0600
+++ configure.ac	2022-01-03 05:25:49.000000000 -0600
@@ -419,17 +419,7 @@
 check_module_rdp="false"
 module_rdp_headers="false"
 
-dnl https://ci.freerdp.com - Nightly FreeRDP master builds 
-if test -d "/opt/freerdp-nightly"; then
-  CPPFLAGS="$CPPFLAGS -I/opt/freerdp-nightly/include"
-  LDFLAGS="$LDFLAGS -L/opt/freerdp-nightly/lib"
-fi
-
 case "$target" in
-  *apple-darwin*)
-    LDFLAGS="$LDFLAGS -L/usr/local/lib/freerdp"
-    RDP_LIBS="-lfreerdp -lwinpr -lfreerdp-client -laudin-client -ldisp-client -lecho-client -lrdpei-client -lrdpgfx-client -ltsmf-client -lcliprdr-client -ldrdynvc-client -lencomsp-client -lrail-client -lrdpdr-client -lrdpsnd-client -lremdesk-client -ldrive-client -lparallel-client -lserial-client -lsmartcard-client"
-    ;;
   *)
     ;;
 esac
@@ -532,12 +522,6 @@
   fi
 fi
 
-case "$target" in
-  *apple-darwin*)
-    AS_MESSAGE([RDP module is currently broken on OS X and disabled by default.])
-    check_module_rdp="false"
-    ;;
-esac
 
 AC_MSG_CHECKING(whether to enable RDP module)
 AC_ARG_ENABLE(module-rdp, 
